\documentclass[10pt]{article}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage[backend=bibtex, style=authoryear]{biblatex}

% \addbibresource{~/Dropbox/TAGreview/PhDreferences.bib}

\textwidth=470pt
\oddsidemargin=0pt
\topmargin=0pt
\headheight=0pt
\textheight=650pt
\headsep=0pt

\title{Statement of Research}


\author{Nicholas Santantonios}
\date{\today}


\newcommand{\gxe}{G$\times$E}
\newcommand{\gxg}{G$\times$G}


\begin{document}

\section*{Introduction}

Alfalfa is the most important forage crop, fourth in production only to maize, soy and wheat. The crop is particularly high in protein, earning it the title of ``Queen of forages''. As feed crops, forage research has been drastically under-funded compared to food crops, but renewed interest in sustainable farming has brought them back into the limelight. Forages increase soil nitrogen, reduce weed populations and generally improve soil health while providing a valuable hay crop. The Cornell forages breeding program has a 100 year legacy of providing forages crop varieties to farmers in the Northeast US, a legacy that I hope to contribute to and extend. 

\section*{Grant 1}

Introgression of beneficial alleles in alfalfa is exceedingly difficult due to high inbreeding depression and the autotetraploid nature of the crop. As an obligate out-crosser, alfalfa must be bred on a population level, where varieties are released as synthetics to avoid inbreeding and take advantage of population level heterosis. This has limited implementation of marker-based selection because large numbers of individuals must be genotyped and inter-mated to avoid inbreeding in future generations. Further complicating the use of markers is which individual(s) should be selected for genotyping to represent a given variety (i.e. population). Single individuals are not representative of the variety as a whole, and genotyping large numbers of individuals from each variety is costly and restrictive.

To implement genomic selection (GS) in alfalfa, I propose a new population-level genomic selection framework (popGS). Tissue from many individuals within a variety or breeding line will be bulked for DNA extraction and genotyped together. This will reduce genotyping costs, while still allowing the standard genomic prediction framework to operate on allele counts in the population as opposed to allele counts within a single individual (See Appendix 1). A sequence based marker platform will need to be built to address the need to estimate allele frequencies of varieties, preferably with the ability to track greater than 2 alleles. PopGS will allow for prediction of additive effects for genetic gain, as well as dominance effects to exploit population level heterosis. The training set will be developed by genotyping varieties and breeding lines from the Cornell forage breeding population and other sources that have been phenotyped within the past decade or so. Moving forward, all new entries into the program will be genotyped in this fashion. New populations will be developed with ``blending crosses'' where individuals from phenotyped lines will be planted in proportions to produce the desired population allele frequencies. This allows popGS to work in a fluid fashion, where new varieties are constructed by mixing the best alleles together in optimal frequencies to produce both high additive and non-additive effects in the resulting populations. 


% A modified rhAmpSeq approach will be built to capture sequences likely to differ at more than one site such that multiple alleles can be tracked.  One approach may include building primer sets that are anchored in exons, but span introns, in order to maximize the likelihood of amplification and sequence differences. Specific motifs? THIS PARAGRAPH NEEDS WORK!

The first grant proposal will be to evaluate the efficacy of population level genomic selection, both theoretically and empirically, and will be submitted (NAFA, USDA, NSF?) by the end of the first year. To provide evidence that this strategy will be successful, I will work with a post doc to write a theory and simulation paper to be submitted within the first year, with the post doc taking first authorship and coauthors to include Kelly Robbins at Cornell and Ian Ray at New Mexico State University. We will also genotype varieties that have been previously phenotyped in order to provide a proof of concept (POC) study as the first milestone of the grant. The diallel consisting of crosses between the nine source populations of alfalfa evaluated at NMSU (Segovia Lerma et al. 2004) is the current target for the POC study, but other trials with large genetic variation may also be considered.

This methodology may also be used for integration of CRISPR mutations into elite germplasm at high frequencies. CRISPR methodology is still in its infancy, but once individual plants can be transformed at all four alleles, crossing into elite germplasm will still be a challenge. Half-sib families will be at best duplex for the CRISPR allele, with further reduction in frequencies due to the inability to intracross or backcross. PopGS can be used to enrich CRISPR allele frequencies until phenotypic evaluation is feasible.

\section*{Grant 2}

Unlike most other agronomic crops, forages are typically harvested multiple times per year, with three to ten cuts a season depending on latitude. Many forages such as alfalfa are also perennial crops, and require 3 to 4 years of phenotyping to evaluate stand persistence. This makes for a heavy phenotypic burden on the breeding program. Root phenotypes are also of particular interest in forage crops, as drought tolerance and nitrogen cycling are highly important for cover crops and field rejuvenation, but are notoriously hard to collect. Root phenotypes will be collected by using the 1 meter core methodology developed by Santantonio and Santantonio (YEAR) to evaluate course and fine root turnover. Tap root morphology will be measured using cores within the drill strip row, while fine root structure will be evaluated from cores sampled between drill strips within each plot. Root phenotypes will be taken twice per year, after the first and last harvests of the season, to determine root growth cycles. 

To mitigate the phenotypic burden of multiple years, cuts and the measuring of root traits, hyper spectral imaging data will be collected from drones throughout each growth cycle to produce high throughput phenotypes (HTP). Genetic correlations of low throughput phenotypes (LTPs) and HTPs will be estimated using a training population and used to build genomic prediction models that incorporate \gxe\ to predict unobserved LTPs. To exploit the time coefficient in the breeders equation, fast cycle selection and blending crosses will be performed on a yearly basis using popGS and available HTPs and LTPs that will be updated yearly from plots in the field. 

The second grant proposal will be incorporation of HTP into a fast cycle forage breeding program, and will be submitted by the end of the first year. The efficacy of HTP selection will be assessed using three selection schemes, where crosses will be made on genotypic predictions (i.e. popGS) of cycle 0 at year 1 (C0Y1), genotype predictions including HTP at year 1, and phenotypic information from year 1 alone. After 1 round of selection, the resulting populations will be planted in the field for evaluation (C1.1). Additional phenotypes from cycle 0, year 2 (C0Y2) and cycle 1, year 1 (C1Y1) will be used to update prediction models for an additional round of selection from both C0 and C1 to produce C1.2 and C2.1 populations, respectively. Three year evaluation of these three generations will be contrasted to determine the merit of HTP assisted popGS. 

\section*{Grants 3 and 4}

Forage crops are only one part of an agronomic biological system which includes soil microorganisms, the animals that feed on the forages and the microbiome of those animals. Other than host pathogen interactions, little attention has been paid to genomic interactions between these organisms (i.e. \gxg), despite an overall notion that they are important. These interactions can be thought of as a special case of the \gxe\ problem, where the covariance of the ``environment'' (e.g. soil microorganisms) can be determined by simply genotyping that ``environment''. Interactions can then be modeled using standard \gxe\ machinery, where the genetic covariance of the biological combinations is a kronecker product of their genetic relationships. Two interacting systems will be targeted for research into \gxg: the interaction between legume and soil microbes, and the interaction between animal and feed. 

As a legume, alfalfa and other legumous forages can form symbiotic relationships with nitrogen fixing bacteria, \emph{Rhizobium}. Unfortunately, the signaling and infection process for nodulation is typically reduced or absent under all but the lowest soil nitrogen levels. Collection, mutagenesis, and transformation of \emph{Rhizobium} may enable the genetic variation necessary for simultaneous \gxg\ selection of host variety and symbiont. Genetic increases in nodulation could allow for the use of less chemical fertilizer, reducing the environmental impact of nitrogen runoff. 

The use of \gxg\ prediction models will allow for targeted selection of host symbiont combinations for testing that are likely to be beneficial. Whole genome expression of both host and symbiont as an intermediate interaction phenotype may also aid in understanding the genetic variation in signaling between the two species through time, under contrasting levels of available nitrogen. This may help us to understand how to select for changes in gene expression to increase the rate of colonization, as well as nitrogenase and leghemaglobin expression, even when some nitrogen is available in the soil. 

% On the other side of the plant, 

There has been considerable research into nutrition of different feeds and how they interact with the microbiome of the rumen in dairy cows. While measures of forage quality are considered, little is known about the effect of different varieties on the animal and its microbiome. I seek to establish a collaboration with the animal nutrition department, local dairy farmers, and animal genetics companies that serve the Northeast to investigate the potential for synergistic forage and animal breeding. Instead of breeding animals independently of their feed, we can start to breed specialized animals to specialized feeds. This is a long term project that will take time to establish the relationships.  While it will take time to establish relationships with animal breeders, dairy farmers and hay farmers, bringing these communities together would set precedent for future integrated breeding operations.

\section*{Personnel}

I would like to hire one post doctoral associate and one graduate student for the first year, with the potential for additional graduate students and post docs for the following year depending on grant funding. The post doc will work with me to write the popGS theory and simulation paper. The graduate student will be responsible for development of the population level genotyping platform using seed from the previously phenotyped population we select for the POC, with guidance from myself and the post doc. This will lead to two companion manuscripts, with the first on the genotyping platform, and the second on the results from the popGS POC study. 

The forage breeding program at Cornell consists of a strong team, whose knowledge and experience will be invaluable to implementing a successful GS breeding program. Additional personnel will be brought on as necessary, including a technician to operate and manage all HTP equipment and activities.

\section*{A forage community}

% One of the challenges is the lack of large phenotypic records for alfalfa in any alfalfa breeding program.

To encourage collaboration, data sharing and a global initiative, a new forage database ($\alpha\alpha$base) be built using either the T3, or cassavabase platform. All tools built by our lab will be made publicly available through the (BRAPI/ galaxy etc?) to allow forage breeding programs throughout the world to make more informed mating decisions. Other programs that produce useful tools for forage breeding will also be encouraged to link them to the $\alpha\alpha$Base through BRAPI.

% Th to encourage public (and perhaps private?) breeding programs to contribute and use data to . We want to implement a genomic prediction tool for alfalfa breeders that will also design idealized mating schemes .

I intend to continue to foster collaborations built by Dr. Don Viands and his forage breeding research team, while hoping to expand them with new research groups, such as that of Animal Nutrition (Cornell), Ian Ray (NMSU), Charlie Brummer (UC Davis), Maria Monteros (Noble Foundation), and Debby Samac (USDA ARS at UMN, and the new forage breeder they are currently looking to hire). 


Finally, I recognize that the forage breeding project currently focuses on many forages other than alfalfa. It is my intent to keep those projects, while incorporating new technologies for genetic gain in these crop. My focus on alfalfa is driven by the need for reliable GS methods in a polyploid crop that must be bred at the population level. I am open to new species as well, especially if a graduate student has a desire to investigate such species.  



% The third grant proposal will be to unify breeding efforts. Genotype by genotype interactions. 

% Field establishment 


% Other long term goals include:

% Collaborate with the Agro-ecology biochemistry.  CRISPR will be used to change both the Rhizobium as well as the plant. 

% Collaboration with the animal nutrition department. sciences


% A technician will be recruited to operate the HTP platform. 

% Implementation of marker-based selection has also lagged because large numbers of individuals must be genotyped to avoid inbreeding depression in future generations. 






\end{document}
